plugin/fuzz/source/dextool/plugin/fuzz/sutxml.d

```diff
@@ -186,7 +186,15 @@ public:
     }
 }
 
+void _debug() {
+    writeln(SutEnvironment("./sut_unittest/namespaces/").getAllNamespaces);
```

Comment:
Tip, there exist a debug functionality i the standard library. It allows runtime configuration of debug level.
```d
import logger = std.experimental.logger;
logger.info(SutEnvironment("./sut_unittest/namespaces/").getAllNamespaces);
```